scripts/sfpshow

```diff
@@ -126,6 +126,15 @@ dom_value_unit_map = {'rx1power': 'dBm', 'rx2power': 'dBm',
                       'tx3power': 'dBm', 'tx4power': 'dBm',
                       'temperature': 'C', 'voltage': 'Volts'}
 
+table_header_info = ['Interface', 'Identifier',
+                    'Connector', 'Vendor Name',
+                    'Model Name', 'Serial Number',
+                    'Nominal Bit Rate(100Mbs)']
+
+table_header_dom = ['Interface', 'Lane Number',
+                    'Temp(C)', 'Voltage(V)',
```

Comment:
I have a concern about the way you handle the temperature, it's not some sensor that monitored per lane, here you added it to each lane, it's kind of misleading.